docs/collect-intro.rst

```diff
@@ -3,24 +3,35 @@ ODK Collect
 
 .. _collect-introduction:
 
-:dfn:`ODK Collect` is an open source Android app that replaces paper forms used in survey-based data gathering. It supports a wide range of question and answer types, and is designed to work well without network connectivity.
+:dfn:`ODK Collect` is an open source Android app for managing and filling out forms. It supports a wide range of question and answer types, and is designed to work well without network connectivity.
```

Comment:
I'd drop open source.

"is designed to work well offline" is more human-friendly.